bandit/core/context.py

```diff
@@ -57,18 +45,19 @@ def call_args(self):
         :return: A list of function args
         '''
         args = []
-        for arg in self._context['call'].args:
-            if hasattr(arg, 'attr'):
-                args.append(arg.attr)
-            else:
-                args.append(self._get_literal_value(arg))
+        if 'call' in self._context and hasattr(self._context['call'], 'args'):
+            for arg in self._context['call'].args:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)